client/lib/helpers/responsiveness.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+class Responsive extends StatelessWidget {
+  final Widget mobile;
+  final Widget? tablet;
+  final Widget desktop;
+
+  const Responsive({
+    Key? key,
+    required this.mobile,
+    this.tablet,
+    required this.desktop,
+  }) : super(key: key);
+
+  static bool isMobile(BuildContext context) =>
+      MediaQuery.of(context).size.width < 850;
+
+  static bool isTablet(BuildContext context) =>
+      MediaQuery.of(context).size.width < 1100 &&
+      MediaQuery.of(context).size.width >= 850;
+
+  static bool isDesktop(BuildContext context) =>
+      MediaQuery.of(context).size.width >= 1100;
+
+  @override
+  Widget build(BuildContext context) {
+    final Size size = MediaQuery.of(context).size;
+    if (size.width >= 1100) {
+      return desktop;
+    } else if (size.width >= 850 && size.width < 1100) {
```

Comment:
Where do those values come from?

Reply:
[here ](https://medium.com/flutter-community/making-cross-platform-flutter-landing-page-responsive-7fffe0655970)

Reply:
Ok, it would be better if you rely on an external library:

https://pub.dev/packages/responsive_builder
https://pub.dev/packages/responsive_sizer
https://pub.dev/packages/adaptive_breakpoints
…

Reply:
Wow! that more easier than my approach , thank you @g123k